go/parquet/pqarrow/schema_test.go

```diff
@@ -36,9 +36,13 @@ func TestGetOriginSchemaBase64(t *testing.T) {
 	origArrSc := arrow.NewSchema([]arrow.Field{
 		{Name: "f1", Type: arrow.BinaryTypes.String, Metadata: md},
 		{Name: "f2", Type: arrow.PrimitiveTypes.Int64, Metadata: md},
+		{Name: "uuid", Type: pqarrow.NewUUIDType(), Metadata: md},
 	}, nil)
 
 	arrSerializedSc := flight.SerializeSchema(origArrSc, memory.DefaultAllocator)
+	if err := arrow.RegisterExtensionType(pqarrow.NewUUIDType()); err != nil {
```

Comment:
don't forget to defer the unregister call. `defer arrow.UnregisterExtensionType("uuid")` it uses the extension name to unregister, so you can either hardcode it or you can do something like:

```go
uuidType := pqarrow.NewUUIDType()
if err := arrow.RegisterExtensionType(uuidType); err != nil {
    t.Fatal(err)
}
defer arrow.UnregisterExtensionType(uuidType.ExtensionName())
```